src/controls/LivePersona/LivePersona.tsx

```diff
@@ -37,7 +37,8 @@ if (isComponentLoaded) {
         hostAppPersonaInfo: {
             PersonaType: 'User'
         },
-        upn:  upn,
+        upn: upn,
```

Comment:
Maybe we should leave both properties to support previous LivePersona versions? In this case this PR will not break existing components that are deployed on Standard Release tenants.